README.md

```diff
@@ -1,335 +1,49 @@
-    print(err)
 ```
+make
+[ ... ]
 
-### Add a signed manifest to a media file or stream
-
-Use a `Builder` to add a manifest to an asset.
-
-```py
-try:
-  # Define a function to sign the claim bytes
-  # In this case we are using a pre-defined sign_ps256 method, passing in our private cert
-  # Normally this cert would be kept safe in some other location
-  def private_sign(data: bytes) -> bytes:
-    return sign_ps256(data, "tests/fixtures/ps256.pem")
-
-  # read our public certs into memory    
-  certs = open(data_dir + "ps256.pub", "rb").read()
-  
-  # Create a signer from the private signer, certs and a time stamp service url
-  signer = create_signer(private_sign, SigningAlg.PS256, certs, "http://timestamp.digicert.com")
-
-  # Define a manifest with thumbnail and an assertion.
-  manifest_json = {
-      "claim_generator_info": [{
-          "name": "python_test",
-          "version": "0.1"
-      }],
-      "title": "Do Not Train Example",
+./dist/go-demo ~/testvids/screenshot-signed.jpg
+{
+  "active_manifest": "urn:uuid:019cfdc5-8d91-4b2a-bb0d-f94afb9badef",
+  "manifests": {
+    "urn:uuid:019cfdc5-8d91-4b2a-bb0d-f94afb9badef": {
+      "claim_generator": "Aquareum c2patool/0.9.6 c2pa-rs/0.33.1",
+      "title": "Video File",
+      "format": "image/jpeg",
+      "instance_id": "xmp:iid:755324ba-0c38-4d9a-b7c1-be8a41035ac6",
       "thumbnail": {
-          "format": "image/jpeg",
-          "identifier": "thumbnail"
+        "format": "image/jpeg",
+        "identifier": "self#jumbf=c2pa.assertions/c2pa.thumbnail.claim.jpeg"
       },
+      "ingredients": [],
       "assertions": [
-      {
-        "label": "c2pa.training-mining",
-        "data": {
-          "entries": {
-            "c2pa.ai_generative_training": { "use": "notAllowed" },
-            "c2pa.ai_inference": { "use": "notAllowed" },
-            "c2pa.ai_training": { "use": "notAllowed" },
-            "c2pa.data_mining": { "use": "notAllowed" }
+        {
+          "label": "c2pa.actions",
+          "data": {
+            "actions": [
+              {
+                "action": "c2pa.published"
+              }
+            ]
           }
         }
-      }
-    ]
-  }
-
-  # Create a builder add a thumbnail resource and an ingredient file.
-  builder = Builder(manifest_json)
-
-  # The uri provided here "thumbnail" must match an identifier in the manifest definition.
-  builder.add_resource_file("thumbnail", "tests/fixtures/A_thumbnail.jpg")
-
-  # Or add the resource from a stream
-  a_thumbnail_jpg_stream = open("tests/fixtures/A_thumbnail.jpg", "rb")
-  builder.add_resource("image/jpeg", a_thumbnail_jpg_stream)
-
-  # Define an ingredient, in this case a parent ingredient named A.jpg, with a thumbnail
-  ingredient_json = {
-    "title": "A.jpg",
-    "relationship": "parentOf", # "parentOf", "componentOf" or "inputTo"
-    "thumbnail": {
-        "identifier": "thumbnail",
-        "format": "image/jpeg"
+      ],
+      "signature_info": {
```

Comment:
signature_info is a read only structure from a signed manifest, you can't set these values when defining an ingredient. The SDK may not complain about it, but it will ignore it. This data comes from the certs and Timestamp authority as part of signing.

Reply:
To be clear, this doesn't have anything to do with my changes, I was just putting the output of the first embedded manifest file I had on hand. Could be anything.

That said, anything wrong with how I created it? I used this `c2patool` command:
```
c2patool --manifest manifest.json screenshot.jpg --output screenshot-signed.jpg
```
And this `manifest.json`:
```
{
  "alg": "ES256",
  "private_key": "/home/iameli/testvids/cert.key",
  "sign_cert": "/home/iameli/testvids/cert.crt",
  "ta_url": "http://timestamp.digicert.com",
  "claim_generator": "Aquareum",
  "title": "Video File",
  "assertions": [
    {
      "label": "c2pa.actions",
      "data": { "actions": [{ "action": "c2pa.published" }] }
    }
  ]
}
```
With a self-signed cert I created using this command (followed by pressing Enter a bunch of times):
```
openssl req \
  -new -x509 -nodes \
  -newkey ec:<(openssl ecparam -name secp384r1) \
  -extensions usr_cert \
  -addext "keyUsage = digitalSignature" \
  -addext "extendedKeyUsage = emailProtection" \
  -keyout cert.key -out cert.crt -days 3650
```
Resulting in the following JPEG: <https://git.aquareum.tv/iameli-streams/random-crap/-/raw/32720c015fbb2c7bd61c80c76a39390b0caaa488/screenshot-signed.jpg>

Everything chill?

Reply:
Whoops, had the wrong link in there, this is the correct signed jpeg: https://git.aquareum.tv/iameli-streams/random-crap/-/raw/dde806417a4540b721f15a269ce46a5f7132d328/screenshot-signed.jpg